Makefile

```diff
@@ -298,8 +298,8 @@ v8:
 jstest: build-addons build-js-native-api-tests build-node-api-tests ## Runs addon tests and JS tests
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=$(BUILDTYPE_LOWER) \
 		--skip-tests=$(CI_SKIP_TESTS) \
-		$(CI_JS_SUITES) \
-		$(CI_NATIVE_SUITES)
+		$(JS_SUITES) \
+		$(NATIVE_SUITES)
```

Comment:
FWIW the reason we split tests into `CI_JS_SUITES` and `CI_NATIVE_SUITES` was that to accommodate the arm-fanned setup we split the tests into `test-ci-native` (requires a build toolchain set up to build the native addons) and `test-ci-js` (which "should not use a native compiler at all" (see [comment](https://github.com/nodejs/node/blob/08cae95e768b72c9f37fa5471e8c2aa568d8f465/Makefile#L516) in Makefile)).

For testing outside of the CI we don't have the same split in Makefile test targets, so you could optionally have a single make variable instead of two.